distrobox-create

```diff
@@ -41,6 +42,7 @@ Options:
 
 	--image/-i:		image to use for the container	default: registry.fedoraproject.org/fedora-toolbox:35
```

Comment:
Remember to remove "default" due to updated line 11.

Reply:
Default is now set at line 113 if no duplicate is set